design/ceph/multus-network.md

```diff
@@ -0,0 +1,110 @@
+# Multus networking integration with Ceph
+
+We have already explored and explained the benefit of multi-homed networking, so this document will not rehearse that but simply focus on the implementation for the Ceph backend.
+If you are interested in learning more about multi-homed networking you can read the [design documentation on that matter](../core/multi-homed-cluster.md).
+
+To make the story short, [Multus](https://github.com/intel/multus-cni) should allow us to get the same performance benefit as `HostNetworking` by increasing the security.
+Using `HostNetworking` results in exposing **all** the network interfaces (the entire stack) of the host inside the container where Multus allows you to pick the one you want.
+Also, this removes the need of privileged containers (required for `HostNetworking`).
+
+## Proposed CRD changed
+
+We already have a `network` CRD property, which looks like:
+
+```yaml
+network:
+  provider:
+  selectors:
+```
+
+We will expand the `selectors` with the following two hardcoded keys:
+
+```yaml
+selectors:
+  public:
+  cluster:
+```
+
+Each selector represents a [`NetworkAttachmentDefinition`](https://github.com/intel/multus-cni/blob/master/doc/quickstart.md#storing-a-configuration-as-a-custom-resource) object in Multus.
+At least, one must be provided and by default they will represent:
+
+- `public`: data daemon public network (binds to `public_network` Ceph option).
+- `cluster`: data daemon cluster network (binds to `cluster_network` Ceph option).
+
+If only `public` is set then `cluster` will take the value of `public`.
```

Comment:
What if only `cluster` is set? Will `public` use a default?

Reply:
No.